docs/apache-airflow/installation.rst

```diff
@@ -38,117 +45,70 @@ Airflow is tested with:
 
 **Note:** SQLite is used in Airflow tests. Do not use it in production. We recommend
 using the latest stable version of SQLite for local development.
 
-Getting Airflow
-'''''''''''''''
-
-Airflow is published as ``apache-airflow`` package in PyPI. Installing it however might be sometimes tricky
-because Airflow is a bit of both a library and application. Libraries usually keep their dependencies open and
-applications usually pin them, but we should do neither and both at the same time. We decided to keep
-our dependencies as open as possible (in ``setup.cfg`` and ``setup.py``) so users can install different
-version of libraries if needed. This means that from time to time plain ``pip install apache-airflow`` will
-not work or will produce unusable Airflow installation.
+Please note that with respect to Python 3 support, Airflow 2.0.0 has been
+tested with Python 3.6, 3.7, and 3.8, but does not yet support Python 3.9.
 
-In order to have repeatable installation, however, starting from **Airflow 1.10.10** and updated in
-**Airflow 1.10.13** we also keep a set of "known-to-be-working" constraint files in the
-``constraints-master``, ``constraints-2-0`` and ``constraints-1-10`` orphan branches.
-Those "known-to-be-working" constraints are per major/minor python version. You can use them as constraint
-files when installing Airflow from PyPI. Note that you have to specify correct Airflow version
-and python versions in the URL.
+Installation tools
+''''''''''''''''''
 
 The official way of installing Airflow is with the ``pip`` tool.
 There was a recent (November 2020) change in resolver, so currently only 20.2.4 version is officially
 supported, although you might have a success with 20.3.3+ version (to be confirmed if all initial
-issues from ``pip`` 20.3.0 release have been fixed in 20.3.3).
+issues from ``pip`` 20.3.0 release have been fixed in 20.3.3). In order to install Airflow you need to
+either downgrade pip to version 20.2.4 ``pip install --upgrade pip==20.2.4`` or, in case you use Pip 20.3, you need to add option
+``--use-deprecated legacy-resolver`` to your pip install command.
 
 While they are some successes with using other tools like `poetry <https://python-poetry.org/>`_ or
 `pip-tools <https://pypi.org/project/pip-tools/>`_, but they do not share the same workflow as
-``pip``- especially when it comes to constraint vs. requirements management.
+``pip`` - especially when it comes to constraint vs. requirements management.
 Installing via ``Poetry`` or ``pip-tools`` is not currently supported. If you wish to install airflow
-using those tools you should use the constraint files described below and convert them to appropriate
+using those tools you should use the :ref:`constraint files <installation:constraints>`  and convert them to appropriate
 format and workflow that your tool requires.
 
-  **Prerequisites**
-
-  On Debian based Linux OS:
```

Comment:
This information looks out of date. We have a section on system requirements below where there are some more libraries.